cli/pkg/commands/root_test.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2023 KStreamer Authors
```

Comment:
Is this copyright assignment correct, or should it be 2024 Open Data Hub Authors, like the other files?